tests/unit/test_image_agnostic_util.py

```diff
@@ -1,42 +1,50 @@
 from unittest.mock import MagicMock, patch
+
 from opendevin.runtime.docker.image_agnostic_util import (
-    generate_dockerfile_content,
     _get_new_image_name,
+    generate_dockerfile_content,
     get_od_sandbox_image,
 )
 
 
 def test_generate_dockerfile_content():
-    base_image = "debian:11"
+    base_image = 'debian:11'
```

Comment:
oh damn the nightmare of single quotes/double quotes starts again... somehow CI is not detecting.

not a problem of your PR, since we default to single quotes.

Reply:
@li-boxuan 
By the way, how should I format the code locally? I'm currently using PyCharm's formatter, but it seems to not match the project's lint rules.

Reply:
GitHub Desktop will do that.

Reply:
PyCharm not enforcing that?

Reply:
Oh we didn't add linting to test folder. That's why. let me create a PR for that.

Reply:
Seems we did. So, that it worked for me.

Reply:
We didn't enforce that in CI - that's the problem.